src/filing/institutions/Header.jsx

```diff
@@ -31,12 +31,15 @@ const InstitutionsHeader = ({ filingPeriod }) => {
       </Alert>
     )
   }
+
+  const lastFilingDay = filingPeriod == '2019' ? '2nd' : '1st'
```

Comment:
```suggestion
  const lastFilingDay = filingPeriod === '2019' ? '2nd' : '1st'
```

filingPeriod is coming through as a String so we may as well avoid the eslint warning about using `==`